src/App.js

```diff
@@ -160,6 +162,20 @@ function Root() {
     }
   }, [isVersionAllowed, wallet]);
 
+  /**
+   * This effect allows navigation to be triggered from background tasks such as sagas.
+   */
+  useEffect(() => {
+    // Ignore this effect if there is no route to navigateTo
+    if (!navigateTo.route) {
+      return;
+    }
+
+    // Navigate to the informed route and reset the navigateTo state property
+    navigate(navigateTo.route, { replace: navigateTo.replace });
+    dispatch(setNavigateTo(''));
```

Comment:
Why an empty string? Why not`{ route: '', replace: false }`;

Or dispatch an action "resetNavigateTo" that sets `navigateTo` to `initialState.navigateTo`

Reply:
Good point. Implemented `resetNavigateTo()` on https://github.com/HathorNetwork/hathor-wallet/pull/573/commits/0ded8d6511bfa75f55ae9e29fdaa063427f80492